Lib/ufo2ft/featureWriters/ast.py

```diff
@@ -85,8 +85,12 @@ def iterClassDefinitions(feaFile, featureTag=None):
 }
 
 
-def makeLookupFlag(name=None, markAttachment=None, markFilteringSet=None):
-    value = 0 if name is None else LOOKUP_FLAGS[name]
+def makeLookupFlag(name=None, flags=None, markAttachment=None, markFilteringSet=None):
+    value = 0
+    if name:
+        value = LOOKUP_FLAGS[name]
+    if flags:
+        value |= sum(LOOKUP_FLAGS[n] for n in flags)
```

Comment:
maybe instead of using `sum` you could use the `|` bitwise operator since these are bit flags.
E.g. `reduce(operator.or_, flags, 0)`